src/System.Runtime.Extensions/src/System.Runtime.Extensions.builds

```diff
@@ -9,14 +9,6 @@
       <OSGroup>Windows_NT</OSGroup>
     </Project>
     <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Unix</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
-      <TargetGroup>netstandard1.5</TargetGroup>
-      <OSGroup>Windows_NT</OSGroup>
-    </Project>
-    <Project Include="System.Runtime.Extensions.csproj">
       <TargetGroup>net462</TargetGroup>
```

Comment:
Running it on win/unix projects first did update the packageIndex.json (https://github.com/jkotas/corefx/commit/07f7023394bf214120a84a3a1bcedda4a9b6d1f7), but the above error is still there when building the root project